### ISSUE TITLE
Enhancement: a11y for font size and localisation language in account settings

### ISSUE THREAD

**Issue by danny-avila** [danny-avila/LibreChat#3770]
### Discussed in https://github.com/danny-avila/LibreChat/discussions/3769

<div type='discussions-op-text'>

<sup>Originally posted by **kukoboris** August 24, 2024</sup>
When I try to select font size and localisation language in the account settings, scrolling in the options selection menu doesn't work (neither mouse nor keyboard)
<img width="688" alt="изображение" src="https://github.com/user-attachments/assets/1d143712-741e-436e-ac18-7a32ad9d7cfc">
<img width="763" alt="изображение" src="https://github.com/user-attachments/assets/d70c567b-57e4-42c4-84f1-a7b364492296">
</div>

**Comment by kukoboris** [comment]
@danny-avila 
Selecting options does not work...

https://github.com/user-attachments/assets/72942a65-69a2-40fb-bb4d-76a3d2933134

**Comment by danny-avila** [comment]
https://github.com/user-attachments/assets/2b88003e-0758-4510-818e-52165f07bf63

it seems to work on my end, not sure what it could be at first glance

**Comment by danny-avila** [comment]
From the styling on your end, it could be an older version

**Comment by kukoboris** [comment]
@danny-avila I use image: ghcr.io/danny-avila/librechat-dev-api:latest. Made update 10 min ago

**Comment by danny-avila** [comment]
> @danny-avila I use image: ghcr.io/danny-avila/librechat-dev-api:latest. Made update 10 min ago

Oh I see, there is something screwy there. I'm on it

**Comment by danny-avila** [comment]
> @danny-avila I use image: ghcr.io/danny-avila/librechat-dev-api:latest. Made update 10 min ago

this might be unique to this build

**Comment by danny-avila** [comment]
@kukoboris the issue is fixed. in the `librechat-dev-api` build, the packages were resolving to a later version, likely of `headlessui/react`, which are unaccounted for.

I could have simply forced the package at the working version, but that would be a bandaid fix, and though the action I took may be disruptive, it's necessary to avoid bugs like this with other packages.

This is a long-standing issue with that particular image, I've decided to align that build closer to the main `Dockerfile` as it maintains package integrity, which prevents a host of `npm` issues leading to unaccounted dependencies.

Will watch out for this as well as write unit tests for these simple dropdowns

**Comment by kukoboris** [comment]
@danny-avila Great! It works now! Thank you very much!
